### PR TITLE
All icons

### PR DESCRIPTION
**Pull request by AP7557**
display icons for all users, as well as update them

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -21,6 +19,6 @@ test("Typeracer renders after starting game", () => {
     userEvent.click(startGameButton);
     // Verify that the typeracer UI is displayed after clicking "Start Game"
     expect(screen.getByText("Begin typing:")).toBeInTheDocument();
-    expect(document.querySelector(".prompt-text")).toBeInTheDocument();
-    expect(document.querySelector(".player-input")).toBeInTheDocument();
+    expect(document.querySelector(".p")).toBeInTheDocument();
+    expect(document.querySelector(".bXYnqS")).toBeInTheDocument();
```
Comment (yusefmustafa) [inline src/App.test.js:23]: It isn't clear what these 2 lines are testing, I would use the same names from before or something similar.

Comment (AP7557) [reply src/App.test.js:23]: those names are default names from styling

File: src/LoginScreen.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ export default function LoginScreen (){
     setPlayerEmail(email)
     changeIsLoggedIn(true);
     let name = response.profileObj.name;
-    socket.emit('login', {name, email});
+    socket.emit('login', {name, email, room:''});
```
Comment (yusefmustafa) [inline src/LoginScreen.js:22]: Do we need to pass a value for `room` here?

Comment (AP7557) [reply src/LoginScreen.js:22]: no i will take it out

**Review by yusefmustafa — APPROVED** [review]

**Pull request merged**